### ISSUE TITLE
[V3 Core] Increase the maximum amount of characters in a single help message

### ISSUE THREAD

**Issue by ghost** [Cog-Creators/Red-DiscordBot#1512]
### Type:

- [X] Suggestion
- [ ] Bug

### Brief description of the problem
Currently, the maximum amount of characters allowed in all fields combined in a single help embed is 1000. [See this code](https://github.com/Cog-Creators/Red-DiscordBot/blob/V3/develop/redbot/core/help_formatter.py#L193).

I think this is way too little and causes the bot to split `[p]help` into multiple messages much too quickly. Instead, I think the limit should be increased to something like 2000. 
Technically, the maximum amount of characters a rich embed can have is 6000, but I can understand that it's not desirable for the embeds to be *that* big (especially when taking mobile users into consideration).
On the contrary, I think a limit of 1000 makes the bot unnecessarily split the fields too quickly. 2000 is a nice middle ground between the two.

TL;DR - Increase the limit of 1000 chars per help embed to something like 2000 chars per help embed.

**Comment by Tobotimus** [comment]
Hi, thanks for the suggestion!

We are going to stick with a default of 1000, but we realise this doesn't suit everyone. So, we plan on resolving this by adding a global setting for the character limit per page, and also the number of pages needed before the help message is sent in a DM.

The interface for these setting will be done in a new `[p]helpset` command, as suggested in #1605.

**Comment by Tobotimus** [comment]
Resolved in #1694 - see `[p]helpset pagecharlimit`
